drud/ddev
# Bump version to bring in separate-nginx-per-cms, fixes #648

## The Problem/Issue/Bug:

https://github.com/drud/docker.nginx-php-fpm-local/pull/51 solves the problem in drud/ddev#648 where D8 doesn't work with redirect module. 

It also aims to fix related problems with TYPO3, and adds a TYPO3-specific config.

This PR just uses the image from that PR. 

It must be updated with an actual release before this can be pulled. 

OP drud/ddev#648
PR we're pulling in from nginx-php-fpm-local: https://github.com/drud/docker.nginx-php-fpm-local/pull/51

## How this PR Solves The Problem:

## Manual Testing Instructions:

* Remove any .ddev in a project that uses redirect module (or TYPO3) and demonstrates the redirect problem.
* ddev config
* ddev start
* Verify that the problem is fixed.
* `ddev ssh` to verify the correct linked nginx config for the apptype in use. This can be tested with other apptypes as well


## Automated Testing Overview:


## Related Issue Link(s):

## Release/Deployment notes:
